sa3ed_app/api/foundedpersonendpoint.py

```diff
@@ -0,0 +1,64 @@
+import frappe
+from frappe import _
+
+@frappe.whitelist(allow_guest=True)
```

Comment:
Add the request type on whitelist method to specify it. like: `@frappe.whitelist(methods=['GET'],allow_guest=True)`